charts/sck-otel/values.yaml

```diff
@@ -191,6 +191,9 @@ journaldLogs:
 # Persistent storage path for storing logs checkpoints
 checkpointPath: "/var/lib/otel_pos"
 
+exclusiveFiltering:
+  enabled: false
```

Comment:
since it is only for pod logs. how about we put it in `containerLogs:` section?

Reply:
The issue https://github.com/splunk/sck-otel/issues/68 describes it as a separate option, do you want to merge it with `containerLogs:` ?

Reply:
yes, i think it makes sense to be under `containerLogs:` 

and on another note,  I have question on the name of `exclusiveFiltering`. 
it sounds like, you exclude some from the whole data. 
but what it will do is "don't ingest anything unless specifically `included`". 
shouldn't it say something like `useIncludeAnnotation` ?